portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,6 +32,8 @@
 import com.google.appengine.api.datastore.FetchOptions;
 import java.text.SimpleDateFormat;  
 import java.util.Date; 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
TODO that is completed should be removed

Reply:
done